tap_clickup/streams.py

```diff
@@ -220,20 +267,26 @@ class TasksStream(ClickUpStream):
 
     name = "task"
     # Date_updated_gt is greater than or equal to not just greater than
-    path = "/team/{team_id}/task"
     primary_keys = ["id"]
     replication_key = "date_updated"
     is_sorted = True
     # ignore_parent_replication_key = True
     schema_filepath = SCHEMAS_DIR / "task.json"
     records_jsonpath = "$.tasks[*]"
     parent_stream_type = TeamsStream
-
     # Need this stub as a hack on _sync to force it to use Partitions
     # Since this is a child stream we want each team_id to create a request for
     # archived:true and archived:false. And we want state to track properly
     partitions = []
-
+    basepath = "/team/{team_id}/task"
+    space_ids_param = "&space_ids=" + "&space_ids=".join(map(str, space_ids)) if space_ids else "" #Dinamically create the spaces path to be passed on API call
```

Comment:
Params should be handled via https://sdk.meltano.com/en/latest/classes/singer_sdk.RESTStream.html#singer_sdk.RESTStream.get_url_params